cmd/vic-machine/create/create.go

```diff
@@ -1355,10 +1394,13 @@ func logArguments(cliContext *cli.Context) {
 		// generic last because it matches everything
 		g := cliContext.Generic(f)
```

Comment:
One reason that I can think of is that we need to explicitly know when we are dealing with a `StringSlice`, so that we can log/append an individual instance of that option for each element in the slice. For output such as this, for example:
```
--insecure-registry=https://registry1
--insecure-registry=https://registry2
--insecure-registry=https://registry3
```
vs.
```
---insecure-registry=[https://registry1 https://registry2 https://registry3]